modules/about-rhde.adoc

```diff
@@ -43,6 +43,11 @@ The latest release notes for each product that is part of {product-title} are av
 ^|*{microshift} Release Status*
 ^|*Supported {microshift} Version&#8594;{microshift} Version Updates*
 
+^|9.4
+^|4.17
+^|Generally Available
+^|4.17.0&#8594;4.17.z and 4.16&#8594;4.17
```

Comment:
I am pretty sure there is a Y-2 scenario that should be included 

4.15 -> 4.17

Reply:
Right! But officially and externally documented, we want to support this only for EUS -> EUS upgrades.

Reply:
Thanks for clarification @DanielFroehlich. Comment can be ignored.